app/modules/user/payout_schedule.rb

```diff
@@ -15,6 +15,8 @@ module User::PayoutSchedule
   def next_payout_date
     return nil if unpaid_balance_cents < minimum_payout_amount_cents
 
+    return Date.current + 1 if payout_frequency == DAILY && Payouts.is_user_payable(self, Date.current, payout_type: Payouts::PAYOUT_TYPE_INSTANT)
```

Comment:
this is excellent. I couldn't figure out an elegant way to do this prior... but now that we have the new `is_user_payable`, this works perfectly.

thanks!